arex-instrumentation-api/src/main/java/io/arex/inst/runtime/context/ArexContext.java

```diff
@@ -151,5 +160,10 @@ public void clear() {
         if (attachments != null) {
             attachments.clear();
         }
+        if (mergeRecordQueue != null) {
+            // async thread merge record (main entry has ended)
+            MergeRecordReplayUtil.recordRemain(this);
```

Comment:
`recordRemain`? Not so easy to understand. Suggested naming adjustments.

Reply:
because the queue need reaches the specified capacity (10)  before record, this is a backup. after the main interface ends, record all the remaining data to ensure that nothing is missed.